src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -2980,16 +2979,19 @@ namespace PackagedTerminalHeatPump {
 			MySizeFlag( PTUnitNum ) = false;
 		}
 
+		RhoAir = StdRhoAir;
```

Comment:
@rraustad It looks like the initial mass flow rates for each speed (calculated at StdRhoAir) are calculated once and used in each timestep regardless of the actual psychrometric state of the air in the simulation. Or is there a correction happening elsewhere?